atlite/wind.py

```diff
@@ -90,3 +101,105 @@ def extrapolate_wind_speed(ds, to_height, from_height=None):
+
+def download_turbineconf(turbine=None, store_locally=True):
+    """Download a windturbine configuration from the OEDB database.
+    
+    Download the configuration of a windturbine model from the OEDB database
+    into the local 'windturbine_dir'.
+    The OEDB database can be viewed here:
+    https://openenergy-platform.org/dataedit/view/supply/turbine_library
+    (2019-07-22)
+    Only one turbine configuration is downloaded at a time, if the 
+    search parameters yield an ambigious result, not data is downloaded.
+    
+    Parameters
+    ----------
+    turbine : dict
+        Search parameters, either provide the turbine model id (takes priority)
+        or a manufacturer and turbine name, e.g. the following are identical:
+        {'id':10}, {'id':10, 'manufacturer':'Unknown', name:'E-53/800'},
+        {'manufacturer':'Enercon', name:'E-53/800'}
+    store_locally : bool
+        (Default: True) Whether the downloaded config should be stored locally
+        in config.windturbine_dir.
+        
+    Returns
+    -------
+    turbineconf : dict
+        The turbine configuration downloaded and stored is also returned as a dict.
+        Has the same format as returned by 'atlite.ressource.get_turbineconf(name)'.
+    """
+    
+    try: 
+        # Get the turbine list
+        OEDB_URL = 'https://openenergy-platform.org/api/v0/schema/supply/tables/turbine_library/rows'
+        result = requests.get(OEDB_URL)
+    except requests.exceptions.RequestException as e:
+        logger.info(f"Connection to OEDB failed with:\n\n{str(e)}")
+        return None
+
+    # Convert JSON to dataframe for easier filtering
+    # Only consider turbines with power curves available
+    df = pd.DataFrame.from_dict(result.json())
+    df = df[df.has_power_curve]
+
+    # Check which information is provided for lookup and proceed accordingly
+    if turbine.get('id'):
+        logger.info(f"Searching for turbine in OEDB with id '{turbine['id']}'.")
+        ds = df[df.id == int(turbine['id'])]
+        
+    else:
+        if not all({isinstance(v, str)
+                    for k,v in turbine.items() if k in ['name', 'manufacturer']}
+                  ):
+            
+            logger.error("'name' and 'manufacturer' must be provided. "
+                         "Alternatively provide a turbine id from OEDB.")
+            return None
+        logger.info(f"Searching turbine power curve in OEDB database using "
+                    f"manufacturer '{turbine['manufacturer']}'"
+                    f"and name '{turbine['name']}'.")
+            
+        ds = df[df.name == turbine['name']]
+        ds = ds[ds.manufacturer == turbine['manufacturer']]
+
+    if len(ds) < 1 :
+        logger.info(f"No turbine found for {turbine}.")
+        return None
+    elif len(ds) > 2 :
+        logger.info(f"Provided information corresponds to more than one turbine. "
+                    f"Provide id to unambiguous lookup. {turbine}.")
```

Comment:
We should show `ds` in the logging message, so that the user can choose.

Reply:
I would presonally argue that this would clutter the log too much (think of between 2-~100 turbines which we list here).
And what to list? The full data? Or only ID + Name + Manu.?

Reply:
Then show:
```python
ds[['id', 'name', 'manufacturer']].head(3)
```